src/components/Layout/ContentSection/RsvDetails.tsx

```diff
@@ -29,6 +33,19 @@ export const RSVDetails: React.FC<Props> = ({ meta, geometry }) => {
               </ButtonLink>
             </div>
           )}
+          <h2 className="mt-6 text-2xl font-extrabold text-gray-900 sm:text-3xl">
+            Projektdaten
+          </h2>
+          <h3 className="mt-6 text-lg font-bold text-gray-900 sm:text-xl">
+            Trassenführung
+          </h3>
+          <p className="text-lg text-gray-500 sm:text-xl">
+            {`${meta.general.from} - ${meta.general.to}`}
+          </p>
+          <h3 className="mt-6 text-lg font-bold text-gray-900 sm:text-xl">
+            Länge
+          </h3>
+          <p className="text-lg text-gray-500 sm:text-xl"> 10 km</p>
```

Comment:
Wenn du ganz fancy sein willst, dann …
```suggestion
          <p className="text-lg text-gray-500 sm:text-xl">10&thinsp;km</p>
```

Das macht (IMO) einen schmalen non breaking space, also klebt das km an die Zahl, so dass es keinen Zeilenumbruch `10 \nkm` geben kann. Aber halt als dünnen Abstand, was typografisch etwas idealer ist. Habe ich aber erst in Safetycheck für %-Werte angefangen zu nutzen; sonst ist ein normaler `&nbsp;` in diesen Fällen die gute Wahl.